test/suites/storage_driver_ceph.sh

```diff
@@ -130,6 +130,24 @@ test_storage_driver_ceph() {
     lxc storage delete "lxdtest-$(basename "${LXD_DIR}")-pool1"
     lxc storage delete "lxdtest-$(basename "${LXD_DIR}")-pool2"
     ceph --cluster "${LXD_CEPH_CLUSTER}" osd pool rm "lxdtest-$(basename "${LXD_DIR}")-existing-osd-pool" "lxdtest-$(basename "${LXD_DIR}")-existing-osd-pool"  --yes-i-really-really-mean-it
+
+
+    # Test that the pool size setting only applies to the created pool, and not any other pools.
+    ceph --cluster "${LXD_CEPH_CLUSTER}" osd pool set ".mgr" size 3
```

Comment:
what does this do?

Reply:
`.mgr` is a [reserved pool](https://docs.ceph.com/en/latest/mgr/administrator/#module-pool) created by ceph. Here we're testing that only the LXD-managed pool corresponding to the config is affected by the config, and not any other pool that may happen to exist.

Reply:
To give some more context, we're basically setting the size of a pre-existing pool that is not managed by LXD, before creating OSD pools with LXD that have explicitly defined sizes.

This is so we can verify only the newly created pools are affected by the config key.

Reply:
Great, thanks, please can you add that as a comment to the test.

Reply:
Done.